lib/provider.go

```diff
@@ -162,6 +162,10 @@ func (p *Provider) getSamlSessionCreds() (sts.Credentials, error) {
 		ProfileARN:      profileARN,
 		SessionDuration: p.SessionDuration,
 		OktaAwsSAMLUrl:  oktaAwsSAMLUrl,
+		MFAConfig: MFAConfig{
+			MFAFactor: p.profiles["okta"]["mfa_factor"],
```

Comment:
Can you think of any way to validate these? I put in the wrong thing and I just get 
```
DEBU[0006] Failed to validate credentials: Failed to authenticate with okta: &errors.errorString{s:"POST https://segment.oktapreview.com/api/v1/authn: 401 Unauthorized"}
```